src/plugins/intel_gna/src/transformations/swap_input_matmul_gna.cpp

```diff
@@ -125,7 +127,11 @@ static void SwapAndTransposeInputs(
     }
 
     if (transpose == nullptr) {
-        new_node = create_transpose(new_node, last_layer_name);
+        auto in_dims = new_node->input(0).get_shape();
+        if (!is_one_dim_shape(in_dims)) {
```

Comment:
See other comment.

Reply:
is_one_dim_shape() is not applicable to CNN layer.